Examples/Io/Root/include/ActsExamples/Io/Root/RootTrajectoryParametersWriter.hpp

```diff
@@ -77,39 +77,63 @@ class RootTrajectoryParametersWriter final
  private:
   Config m_cfg;             ///< The config class
   std::mutex m_writeMutex;  ///< Mutex used to protect multi-threaded writes
-  TFile* m_outputFile{nullptr};   ///< The output file
-  TTree* m_outputTree{nullptr};   ///< The output tree
-  unsigned int m_eventNr{0};      ///< the event number
-  unsigned int m_multiTrajNr{0};  ///< the multi-trajectory number
-  unsigned int m_subTrajNr{0};  ///< the multi-trajectory sub-trajectory number
+  TFile* m_outputFile{nullptr};  ///< The output file
+  TTree* m_outputTree{nullptr};  ///< The output tree
+  unsigned int m_eventNr{0};     ///< The event number
+  std::vector<unsigned int>
+      m_multiTrajNr;  ///< The multi-trajectory numbers in event
+  std::vector<unsigned int>
+      m_subTrajNr;  ///< The multi-trajectory sub-trajectory number in event
 
-  unsigned long m_t_barcode{0};  ///< Truth particle barcode
-  int m_t_charge{0};             ///< Truth particle charge
-  float m_t_time{0};             ///< Truth particle time
-  float m_t_vx{-99.};            ///< Truth particle vertex x
-  float m_t_vy{-99.};            ///< Truth particle vertex y
-  float m_t_vz{-99.};            ///< Truth particle vertex z
-  float m_t_px{-99.};            ///< Truth particle initial momentum px
-  float m_t_py{-99.};            ///< Truth particle initial momentum py
-  float m_t_pz{-99.};            ///< Truth particle initial momentum pz
-  float m_t_theta{-99.};         ///< Truth particle initial momentum theta
-  float m_t_phi{-99.};           ///< Truth particle initial momentum phi
-  float m_t_pT{-99.};            ///< Truth particle initial momentum pT
-  float m_t_eta{-99.};           ///< Truth particle initial momentum eta
+  std::vector<unsigned long>
```

Comment:
The description of "Each entry in the TTree corresponds to fitted track parameters of one
/// trajectory" above is not correct any more. You might want to fix that.

Reply:
Done.